From: Daniel Leidert <dleidert@debian.org>
Date: Thu, 26 Mar 2020 03:14:58 +0100
Subject: Fix minitest spec tests

---
 test/test_server.rb | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/test/test_server.rb b/test/test_server.rb
index 3b4bbbb..bb21f1c 100644
--- a/test/test_server.rb
+++ b/test/test_server.rb
@@ -90,10 +90,10 @@ describe Dalli::Server do
     end
 
     it 'throws an exception if the hostname cannot be parsed' do
-      lambda { Dalli::Server.new('[]') }.must_raise Dalli::DalliError
-      lambda { Dalli::Server.new('my.fqdn.com:') }.must_raise Dalli::DalliError
-      lambda { Dalli::Server.new('my.fqdn.com:11212,:2') }.must_raise Dalli::DalliError
-      lambda { Dalli::Server.new('my.fqdn.com:11212:abc') }.must_raise Dalli::DalliError
+      assert_raises(Dalli::DalliError){ Dalli::Server.new('[]') }
+      assert_raises(Dalli::DalliError){ Dalli::Server.new('my.fqdn.com:') }
+      assert_raises(Dalli::DalliError){ Dalli::Server.new('my.fqdn.com:11212,:2') }
+      assert_raises(Dalli::DalliError){ Dalli::Server.new('my.fqdn.com:11212:abc') }
     end
   end
 
@@ -141,9 +141,9 @@ describe Dalli::Server do
       s = Dalli::Server.new('127.0.0.1', :error_when_over_max_size => true)
       value = OpenStruct.new(:bytesize => 1_048_577)
 
-      lambda do
+      assert_raises(Dalli::ValueOverMaxSize) do
         s.send(:guard_max_value, :foo, value)
-      end.must_raise Dalli::ValueOverMaxSize
+      end
     end
   end
 
